packages/scripts/src/createPackage.ts

```diff
@@ -41,34 +43,31 @@ export async function createPackage(packageName: string, packageDescription?: st
 	await writeFile(`package.json`, JSON.stringify(packageJSON, null, 2));
 
 	// Update cliff.toml
-	const cliffTOML = (await readFile(join('..', 'scripts/src/template/cliff.toml'), 'utf8')).replace(
-		'{name}',
-		packageName,
-	);
+	const cliffTOML = (await readFile(join(templateDir, 'cliff.toml'), 'utf8')).replace('{name}', packageName);
 
 	await writeFile('cliff.toml', cliffTOML);
 
 	// Update .cliff-jumperrc.json
-	const newCliffJumperJSON = { ...cliffJumperJSON, name: packageName, packagePath: `packages/${packageName}` };
+	const newCliffJumperJSON = { ...cliffJumperJSON, name: packageName, packagePath: packageDir };
 
 	await writeFile('.cliff-jumperrc.json', JSON.stringify(newCliffJumperJSON, null, 2));
 
 	// Move to github directory
-	chdir('../../.github');
+	chdir(join('..', '..', '.github'));
 
 	const labelsYAML = parseYAML(await readFile('labels.yml', 'utf8')) as LabelerData[];
 	labelsYAML.push({ name: `packages:${packageName}`, color: 'fbca04' });
 
 	await writeFile('labels.yml', stringifyYAML(labelsYAML));
 
 	const labelerYAML = parseYAML(await readFile('labeler.yml', 'utf8')) as Record<string, string[]>;
-	labelerYAML[`packages/${packageName}`] = [`packages:${packageName}/*`, `packages:${packageName}/**/*`];
```

Comment:
This will result on a label named `packages\\${packageName}` on Windows. Please revert this line.